dev-haskell/ed25519/ed25519-0.0.5.0.ebuild

```diff
@@ -1,41 +1,33 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
-# ebuild generated by hackport 0.4.6.9999
+# ebuild generated by hackport 0.7.2.2.9999
 #hackport: flags: +no-donna,+test-doctests,+test-properties,-test-hlint
 
-CABAL_FEATURES="lib profile haddock hoogle hscolour" # Broken test-suite: needs update
+CABAL_HACKAGE_REVISION=5
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour" # test-suite"
 inherit haskell-cabal
 
 DESCRIPTION="Ed25519 cryptographic signatures"
-HOMEPAGE="http://thoughtpolice.github.com/hs-ed25519"
-SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+HOMEPAGE="https://thoughtpolice.github.com/hs-ed25519"
 
 LICENSE="MIT"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
-RESTRICT=test # fails on QC-2.10 as: The constructor ‘Failure’ should have 11 arguments, but has been given 10
+RESTRICT=test # bitrotten test suite
 
-RDEPEND=">=dev-lang/ghc-7.4.1:=
+RDEPEND=">=dev-lang/ghc-8.4.3:=
 "
 DEPEND="${RDEPEND}
-	>=dev-haskell/cabal-1.10
+	>=dev-haskell/cabal-2.2.0.1
+	test? ( >=dev-haskell/doctest-0.10 <dev-haskell/doctest-0.12
+		>=dev-haskell/quickcheck-2.4 <dev-haskell/quickcheck-2.9 )
```

Comment:
I think you should comment out these test deps since they will not be used (due to `RESTRICT=test`).

@solpeth Any thoughts on this?

Reply:
This looks like what is causing the `pkgcheck` error (`test?` but no `test` USE flag).

Reply:
You could also leave `CABAL_FEATURES="test-suite"` but I like to comment that out and then comment out any `test?` block under `DEPEND`. I've seen other ebuilds do it differently, though.